netlify/util/getFindingsData.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from "node-fetch";
+import { AwardFinding } from "../../types/finding";
+
+/**
+ * @async
+ * @returns {Promise<[AwardFinding]>}
+ * @throws Various errors if data cannot be fetched or processed.
+ */
+const getApiFindingsData = async () => {
+  const res = await fetch(`${process.env.C4_API_URL}/api/v0/getFindings`, {
+    method: "POST",
+    body: JSON.stringify({ token: process.env.C4_API_TOKEN }),
+  });
+  if (res.status !== 200) {
+    console.log(await res.text());
```

Comment:
```suggestion
```

Reply:
I would actually add the result of `res.text()` to the error message you're throwing. We don't want to obfuscate the error and lose the ability to track what went wrong.